docs/user/extract-images.md

```diff
@@ -19,3 +19,26 @@ for image_file_object in page.images:
         fp.write(image_file_object.data)
         count += 1
 ```
+
+# Other images
+
+Some other objects can contain images, such as stamp annotations.
+
+For example, this document contains such stamps:
+
+[test_stamp.pdf](https://github.com/user-attachments/files/15751424/test_stamp.pdf)
```

Comment:
I would avoid adding a new paragraph here and add the link to the same paragraph as the introductory text.